### PR TITLE
Enable VCL snippets

### PR DESCRIPTION
**Pull request by boris-jarkowicz**
@fstehle @skrings PTAL

7 tests failed locally. they might fail on Travis, too. We'll see … :-|

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -145,8 +157,14 @@ $ ansible-playbook -i localhost, fastly.yml
 ## Development
 
 ### Running the tests
-```
-FASTLY_API_KEY=some_secret python -m unittest discover tests
+
+```bash
+# prepare the environment ...
+$ printf '[defaults]\nroles_path=../' >ansible.cfg
+$ ln -snf $PWD ../Jimdo.fastly
```
Comment (fstehle) [inline README.md]: I don't think this is necessary, the unit tests don't use ansible at all.

File: library/fastly_service.py [hunk 2]
```diff
@@ -327,6 +342,24 @@ def __init__(self, config, validate_choices):
         self.status = self.read_config(config, validate_choices, 'status')
 
 
+class FastlyVclSnippets(FastlyObject):
```
Comment (fstehle) [inline library/fastly_service.py]: This should be `FastlyVclSnippet`.

**Comment by boris-jarkowicz** [comment]
@fstehle you can take a look again, although i am not sure that setting up the services works correctly, because only one shows up in fastly, but when curling the API with the service ID's, i am getting the correct service back. It just doesn't show up in fastly.

File: library/fastly_service.py [hunk 3]
```diff
@@ -107,6 +122,7 @@
 
 import httplib
 import urllib
+import json
```
Comment (fstehle) [inline library/fastly_service.py]: This import is unused

File: tests/test_fastly_vcl_snippets.py [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+
+        self.assertEqual(service.active_version.configuration.snippets[0].name, self.VCL_SNIPPETS_NAMES[3])
+        self.assertEqual(service.active_version.configuration.snippets[0].dynamic, 0)
+        self.assertEqual(service.active_version.configuration.snippets[0].type, 'error')
+        self.assertEqual(service.active_version.configuration.snippets[0].content, content)
+        self.assertEqual(service.active_version.configuration, configuration)
+
+        active_version_number = service.active_version.number
+        service = self.enforcer.apply_configuration(self.FASTLY_TEST_SERVICE, configuration).service
+        self.assertEqual(service.active_version.number, active_version_number)
```
Comment (fstehle) [inline tests/test_fastly_vcl_snippets.py]: One test should be enough as these tests all test the same flow.

File: README.md [hunk 5]
```diff
@@ -144,9 +156,8 @@ $ ansible-playbook -i localhost, fastly.yml
 
 ## Development
 
-### Running the tests
-```
-FASTLY_API_KEY=some_secret python -m unittest discover tests
+# do it!!
+$ FASTLY_API_KEY=some_secret python -m unittest discover tests
```
Comment (fstehle) [inline README.md]: What is this?

**Comment by boris-jarkowicz** [comment]
PTAL again @fstehle

**Pull request merged**